### PR TITLE
60 deploy shared as an npm package

### PR DESCRIPTION
**Pull request by ArtemKech**
# /shared deployed as NPM package!

Now [`ai-daily`](https://www.npmjs.com/package/ai-daily?activeTab=readme) is available on **_npm_** 

# Changes:

### Root:
* Added `.npmignore` file so I can just deploy `/shared`

### API & Client:
* Updated all imports with: `ai-daily`
* Added new package to `package.json`

### Shared & NPM page:
* Updated `README` file with code usage examples

# Usage:

Install with:
```shell
npm i ai-daily
```
And import like:
```typescript
import { Article } from 'ai-daily';
```
# Important note:
Please, let me know if you have an account with **_npm_** so I can add you guys as **collaborators**!

### CODE REVIEW

File: shared/README.md
Comment (AardhynLavender) [inline shared/README.md:1]: Can we replace this with `JSDoc` comments above each method and type. That way, our editors intellisense can display the documentation when we use the library.

**Review by AardhynLavender — CHANGES_REQUESTED** [review]
Looks good, but... Can you make sure there are no duplicated types and functions between the `api` and `client`... I think there are some lying around the client app.

**Review by gpseal — APPROVED** [review]
Great idea, didn't realise this could be done, adds an extra layer to the app, good job

File: shared/README.md
Comment (ArtemKech) [reply shared/README.md:1]: Yeah, this is a really good idea. I will add that and merge it

**Pull request merged**